scripts/symlink.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
I think we should call this file "deps-stage.js" or something like that (keeping in mind that the constants file will have a similar prefix). It's weird that we call it "symlink.js" and replace all the symlink bits. The functionality here is to just provide options for moving the library around.

Reply:
Done. Named it "deps-stage.js"